### PR TITLE
fix(formatters): Ensure auto-unit conversion works also for negative values.

### PR DESCRIPTION
**Pull request by raymond-shanley-dynatrace**
Negative values of same magnitude should have same auto-unit conversion as equivalent positive values.

### CODE REVIEW

**Review by ffriedl89 — APPROVED** [review]
Code change looks good. Please update the security issues in a separate PR :)

**Review by Sherif-Elhefnawy — APPROVED** [review]

**Review by thomaspink — APPROVED** [review]

**Comment by Sherif-Elhefnawy** [comment]
Already cherry-picked into:
10.x: f0a527886227b5dc3ce56bf4fe1ace95ef3f6cdb
10.7.x: bb1a536137582b7a32b9559638f812ef4e8bbc13

waiting on approval from Fabian again due to rebase

**Review by ffriedl89 — APPROVED** [review]

**Pull request merged**
